base/test/nvtransform_tests.cpp

```diff
@@ -126,4 +127,48 @@ BOOST_AUTO_TEST_CASE(fdtest)
 	DMAUtils::freeCudaPtr(eglInImage,&pInResource, eglDisplay);
 }
 
+BOOST_AUTO_TEST_CASE(crop, *boost::unit_test::disabled())
```

Comment:
Please add similar tests with FileReader, if these tests remain "disabled" we will ahve regressions.